package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "zen-audio-player-electron",
+  "version": "0.1.0",
+  "main": "main.js",
+  "dependencies": {
+    "electron-prebuilt": "^1.2.7"
+    },
+   "scripts": {
+    "lint" : "eslint *.js", 
+    "lintinstall" : "npm install eslint",
```

Comment:
This shouldn't be necessary. If Travis fails without it, add `npm install` to the Travis config and remove this script entry